data_safe_haven/__about__.py

```diff
@@ -0,0 +1 @@
+__version__ = "v0.4.0"
```

Comment:
At the moment this is in `pyproject.toml`. Is that a standard location? If we also need it in `__about__.py` we can do

```
import pkg_resources
__version__ = pkg_resources.get_distribution("data-safe-haven").version
```

Reply:
See eg (https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package) and (https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#declaring-project-metadata) .

Reply:
In `pyproject.toml` the version is now dynamic metadata

https://github.com/alan-turing-institute/data-safe-haven/blob/f01446eabc6237930d10e49f61d59d1687887e9d/pyproject.toml#L7

sourced from `__about__.py`

https://github.com/alan-turing-institute/data-safe-haven/blob/f01446eabc6237930d10e49f61d59d1687887e9d/pyproject.toml#L52-L53

So it is only defined in one place (and you can use hatch to update the version).
This is [#1 of the options here](https://packaging.python.org/en/latest/guides/single-sourcing-package-version/), just for hatch rather than setuptools.

I think the things you have linked won't work as they rely on setuptools, which isn't being used here.

Reply:
We can import (or move `version` to `__init__.py`, maybe renamed `__version__`) to get the value in the code.

Reply:
Looks like a relatively common standard is to have `version.py` that contains:

```
__version_info__ = ('1', '2', '3')
__version__ = '.'.join(__version_info__)
```

then `__init__.py` has `from .version import __version__, __version_info__`.

Presumably this will still work with `pyproject.toml` dynamic while providing the PEP-recommended `<module>.__version__` string? As far as I can tell, `__about__.py` is not specified anywhere.

Reply:
Fixed in 18cba63d